installation/GRCh38_related_files/calc_gc_content.py

```diff
@@ -0,0 +1,58 @@
+import sys
```

Comment:
Maybe rename the file. You're calculating a GC-content BED track. E.g. "calc_gc_content_bed.py"